tools/corosync-notifyd.c

```diff
@@ -282,7 +283,14 @@ static void _cs_cmap_members_key_changed (
 	if (res) {
 		strncpy(nodename, open_bracket, CS_MAX_NAME_LENGTH-1);
 	}
```

Comment:
I think previous 4 lines should be deleted, because currently it looks like:

```c
res = _cs_ip_to_hostname(open_bracket, nodename);
if (res) {
  strncpy(nodename, open_bracket, CS_MAX_NAME_LENGTH-1);
}
If(conf[CS_NTF_NODNS]) {
  strncpy(nodename, open_bracket, CS_MAX_NAME_LENGTH-1);
} else {
  res = _cs_ip_to_hostname(open_bracket, nodename);
  ...
```

Reply:
Fixed in a new commit. Was manually cherry-picking from my working repo and forgot to delete those lines.